conans/client/command.py

```diff
@@ -735,7 +735,8 @@ def run(self, *args):
             self._user_io.out.error(msg)
         except Exception as exc:
             import traceback
-            print(traceback.format_exc())
+            # print(traceback.format_exc())
```

Comment:
It is in develop because I always use it developing and probably I committed it...